examples/sasl-gssapi.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This is a simple example demonstrating how to configure clients to communicate with a Kerberized Kafka cluster.
+# Wire encryption(TLS) is also covered in this example.
+#
+# See `Configuring GSSAPI` in the confluent docs for instructions on setting up a Kerberized Kafka Cluster.
+#
+# https://docs.confluent.io/current/kafka/authentication_sasl/authentication_sasl_gssapi.html
+#
+# See also `Encryption with SSL` for instructions on setting up TLS communication with a Kafka Cluster.
+#
+# https://docs.confluent.io/current/kafka/encryption.html
+#
+
+import uuid
+
+import certifi
+
+from confluent_kafka import Producer, Consumer
+
+
+def errored(err):
+    """Global/Generic error report callback served upon calling `flush()` or `poll()`."""
+    print('Error: %s' % err)
+
+
+p = Producer({
+    'bootstrap.servers': 'localhost:9093',
```

Comment:
Dont hard-configure bootstraps or group.ids, this needs to be runnable in the dev's environment.
They should be passed on the command line, at least the brokers as long as the group.id is very specific to this example and doesnt risk clashing with existing stuff.